imkar/scattering/diffuse.py

```diff
@@ -0,0 +1,178 @@
+import pyfar as pf
+import numpy as np
+import numbers
+
+
+def diffuse(reverberation_time, speed_of_sound, air_attenuation,
+            calculation_method='ISO', scale_factor=5,
+            sample_area=0.5026548, baseplate_area=0.58088,
+            surface_room=9.05, volume_room=1.67):
+    """
+    This function calculates the diffuse scattering coefficient after the ISO
+    17497-1 method. It uses four reverberation times measured under different
+    conditions [#]_.
+
+    Parameters
+    ----------
+    reverberation_time : pf.FrequencyData
+        Frequency object containing the reverbaration times calculated during
+        the four different measurements. If the measurement is performed at
+        multiple positions, the mean value for every condition of the
+        measurements is taken. Its cshape has to be (..., 4).
+
+    speed_of_sound : array, float
+        Array containing the speed of sound in m/s for every measurement
+        condition calculated by the temperature and air humidity during
+        measurement time.
+
+    air_attenuation : array, pf.FrequencyData
+        Matrix or FrequencyData containing the air attenuation in 1/m for every
+        measurement condition calculated by temperature and air humidity during
+        measurement time. Its cshape has to be (..., 4).
+
+    calculation_method : 'ISO','Eyring' or 'Sabine'
```

Comment:
```suggestion
    calculation_method : string
```
describe each option as it is done for distance_measure in find_nearest
https://pyfar.readthedocs.io/en/stable/classes/pyfar.coordinates.html#pyfar.classes.coordinates.Coordinates.find_nearest